apphub/image_classification/lenet_mnist/pytorch/lenet_mnist.py

```diff
@@ -6,35 +20,14 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-from torch.utils.data import DataLoader, Dataset
 
 import fastestimator as fe
+from fastestimator.pipeline import Pipeline, NumpyDataset
+from fastestimator.op.numpyop import Minmax, ExpandDims
 from fastestimator.op.tensorop.loss import CrossEntropy
 from fastestimator.op.tensorop.model import ModelOp, UpdateOp
 from fastestimator.trace.metric import Accuracy
 
-class MnistDataset(Dataset):
```

Comment:
can you create a new file in the pytorch folder instead of over-writing the old one?   we need a file that showcases torch data loader in pipeline for now.

Reply:
realistically we're not going to have 4 different examples just to show off these permutations... I think in app hub we probably don't want any examples that aren't using our pipeline since we want users to do it our way. That would be something better covered in a tutorial

Reply:
yes, we will move it to tutorial later in the future when we re-write the tutorials. For now, let's have two files for this example only.

Reply:
added the extra files for now